packages/flutter/lib/src/material/pickers/date_picker_header.dart

```diff
@@ -142,7 +142,7 @@ class DatePickerHeader extends StatelessWidget {
                  children: <Widget>[
                    const SizedBox(height: 16),
                    Flexible(child: help),
-                   const SizedBox(height: 38),
+                   const SizedBox(height: 30),
```

Comment:
I think the problem here is that in this locale the font sizes appear to be larger than ISO Latin fonts, which is causing some layout issues. Instead of changing this constant, a better fix would be to have this spacing SizedBox be the Flexible child instead of the help text. That way if there isn't enough room due to font size issues, the slack will be taken from this spacing widget instead of the help text. I am thinking something like:

```dart
const SizedBox(height: 16),
child: help,
const Flexible(child: SizedBox(height: 38)),
```

Reply:
@darrenaustin can you review my PR again please